types/imgix-api-test.ts

```diff
@@ -25,7 +26,24 @@ ix.request(`sources/upload/${SOURCE_ID}/image.jpg`, {
 });
 
 // $ExpectType Promise<RequestResponse | RequestError>
-ix.request(`sources`, {
+ix.request('sources', {
   method: 'POST',
   body: BODY_JSON,
 });
+
+// $ExpectType Promise<void>
+ix.request('sources').then((response) => {
+  response = response as RequestResponse;
```

Comment:
This one's kind of iffy and might as well be excluded. Curious if there are any suggestions on how to do this better.

Reply:
I think this is okay, but I would just drop line 34 since it's not actually testing anything